android/app/build.gradle

```diff
@@ -137,13 +143,17 @@ android {
 }
 
 dependencies {
-    compile project(':react-native-svg')
-    compile project(':react-native-vector-icons')
-    compile project(':react-native-maps')
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
-    implementation "com.trinerdis.skypicker:react-native-bridge:0.1-SNAPSHOT"
+    implementation project(':react-native-svg')
```

Comment:
How are these change related to the logging library?

Reply:
Gradle build tool has been upgraded. Compile is deprecated, implementation is the replacement.

Reply:
Alright, this explanation is enough for me... :)